dell/inspiron/14-5420/README.md

```diff
@@ -0,0 +1,13 @@
+# Dell Inspiron 14 5420
+
+### Hardware
+- CPU: Intel® Core™ i7-1260P Processor
+- GPU: Intel® Iris® Xe Graphics
+
+### Other Info
+`fwupd` is used to receive firmware updates from vendors. This device is in its [supported devices list](https://fwupd.org/lvfs/devices/)
+
+`thermald` is used to achieve better thermal behaviour
+
+##### TLP
+TLP is a battery power optimizer for laptops. You may enable it with `services.tlp.enable = true;`, however, please note that it may conflict with GNOME's power management. Enabling TLP may require additional configurations.
```

Comment:
We do this already in ../../../common/pc/laptop if the gnome power manager is not present.

Reply:
That is now fixed as well